packages/playground/src/dashboard/components/public_config.vue

```diff
@@ -139,20 +139,20 @@
           <v-toolbar color="primary" dark class="custom-toolbar">
             <p class="mb-5">Remove Public Config</p>
           </v-toolbar>
-          <v-card-text> Are you certain you want to remove this node's public config? </v-card-text>
+          <v-card-text>Remove this node's public config? </v-card-text>
```

Comment:
same as the comment below, it shouldn't start with remove, imo it should be the same as before or rephrased to `Do you want to remove this node's public config?`